uint8 blow_off_mode		# 0/1/2 - check manual
uint8 soft_start		# 0/1
uint8 out2_function		# 0/1
uint8 out3_function		# 0/1
uint8 signal_type_input		# 0/1
uint8 signal_type_output	# 0/1
uint8 vacuum_unit		# 0/1/2/3
uint8 output_filter		# 0/1/2/3
uint8 eco_mode			# 0/1
uint8 display_rotation		# 0/1
